support/iosConstants.ts

```diff
@@ -1,4 +1,4 @@
-export const IPHONEOS_DEPLOYMENT_TARGET = "10.0";
+export const IPHONEOS_DEPLOYMENT_TARGET = "11.0";
```

Comment:
Why bump the default to iOS 11 for the NSE? The NSE works all the way back to iOS 10.

Reply:
The Podfile is generated automatically by Expo. Depending on the Expo version they're using, this value will be either 10, 11, or 12. Newest Expo versions will set this value to 12.0. Changing the default to 11 simply decreases the possibility that there will be a mismatch between the deployment target version and the version in their Podfile.

If they're using the newest Expo version, they will still need to customize the deployment target since the target default would be 11 and the Podfile version is 12.0.

If they're using an old-ish Expo version, the versions will more likely match 11.0 with 11.0 -- meaning they won't need to pass in a custom deployment target version.

If they're using a _very_ _very_ old version of Expo (unlikely), they will have a higher deployment target (11.0) than their Podfile (e.g: 10.0). However, this should still work fine.